dsl-reference.md

```diff
@@ -1397,7 +1421,7 @@ An event filter is a mechanism used to selectively process or handle events base
 
 | Property | Type | Required | Description |
 |----------|:----:|:--------:|-------------|
-| with | `object` | `yes` | A name/value mapping of the attributes filtered events must define. Supports both regular expressions and runtime expressions.  |
+| with | [`eventProperties`](#event-properties) | `yes` | A name/value mapping of the attributes filtered events must define. Supports both regular expressions and runtime expressions.  |
```

Comment:
So this type in the schema can be the new definition introduced by @matthias-pichler-warrify and also a [pattern](https://json-schema.org/understanding-json-schema/reference/string#regexp)? Maybe it's worth updating the schema?